common/credCache_bsd.go

```diff
@@ -0,0 +1,128 @@
+// +build freebsd
```

Comment:
Here's a suggestion. Instead of specifically only supporting freeBSD with this dummy cred cache, how about we make it a general fallback cred cache?

This'd allow users on other OSes, even other types of BSD (such as netBSD and such) to build azcopy as well, in theory.

Reply:
I am supportive of this idea, but unsure how to implement it.  I am only vaguely familiar with this project's source code and have a "beginner" level familiarity with Go at this point.

For now, it seems like it would be quite easy to use this as-is and just add "netbsd" and others to the `+build` line as-needed.  If you have a better concrete set of changes in mind for me, or want to take the proposal and run with it, please let me know.  I'm happy to hand it off in some form.  Thanks!

Reply:
I think we'll leave any wider usage to later.